src/models/room-member.js

```diff
@@ -34,6 +34,7 @@ const utils = require("../utils");
  * @prop {string} userId The user ID of this member.
  * @prop {boolean} typing True if the room member is currently typing.
  * @prop {string} name The human-readable name for this room member.
```

Comment:
you might say "disambiguated using the user-id if necessary" here, just to help clarify